command/operator_members.go

```diff
@@ -70,9 +71,33 @@ func (c *OperatorMembersCommand) Run(args []string) int {
 
 	switch Format(c.UI) {
 	case "table":
-		out := []string{"Host Name | API Address | Cluster Address | ActiveNode | Last Echo"}
+		showUpgradeVersion := false
+		out := make([]string, 0)
+		cols := []string{"Host Name", "API Address", "Cluster Address", "Active Node", "Version"}
```

Comment:
Why not include redundancy zone?  Seems like if we're going to add upgrade version we should add that too, no?

Reply:
My main reason for adding upgrade version was so that you could monitor the version of nodes during an upgrade. I didn't think redundancy zones would be useful for this particular command but I can add it.

Reply:
Also, this is one more field to conditionally show/hide. I doubt we want to clutter the output of the OSS version of this command with fields that will always be blank, right?

Reply:
I'd be fine with blank fields.  Sometimes people use the oss version on the cli to talk to ent servers.  And the data will be present in the other formats anyway.